src/components/NavBar/NavBar.tsx

```diff
@@ -13,13 +13,13 @@ interface NavBarPropsType {
   mini: Boolean;
 }
 const NavBar = ({ toggleMainWidth, mini }: NavBarPropsType) => {
-  const [sideBarId, setSideBarId] = useState("sidebar-min");
```

Comment:
Why changing the id of your component (which is highly risky and could lead to css issues), if you can add a second css class and change it dynamically?

In you css file, change the ids that control the width to classes, and then combine `sidebar` and `sideBarId` toghther (Read about [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals))

Reply:
Added a second class

Reply:
Kapara on your life